<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HisDataSearch</class>
 <widget class="QWidget" name="HisDataSearch">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="lbTitle">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>30</y>
     <width>271</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>搜                                索</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leYear">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>190</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="leMinute">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>270</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbHour">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>260</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>时</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leSecond">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>270</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="leHour">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>270</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbMonth">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>180</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>月</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbMinute">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>260</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>分</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbDay">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>180</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>日</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbYear">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>180</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>年</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbSearch">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>210</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>搜索</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>160</y>
     <width>641</width>
     <height>171</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLineEdit" name="leMonth">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>30</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QLineEdit" name="leDay">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>190</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbSecond">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>260</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>秒</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbExit">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>370</y>
     <width>98</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbDataTime">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>110</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>当前日期时间</string>
   </property>
  </widget>
  <zorder>frame</zorder>
  <zorder>lbTitle</zorder>
  <zorder>leYear</zorder>
  <zorder>leMinute</zorder>
  <zorder>lbHour</zorder>
  <zorder>leSecond</zorder>
  <zorder>leHour</zorder>
  <zorder>lbMonth</zorder>
  <zorder>lbMinute</zorder>
  <zorder>lbDay</zorder>
  <zorder>lbYear</zorder>
  <zorder>pbSearch</zorder>
  <zorder>leDay</zorder>
  <zorder>lbSecond</zorder>
  <zorder>pbExit</zorder>
  <zorder>lbDataTime</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
